lib/lightning_web/live/job_live/inspector_form_component.ex

```diff
@@ -188,7 +194,7 @@ defmodule LightningWeb.JobLive.InspectorFormComponent do
               phx-hook="Editor"
               phx-update="ignore"
               id="editor-component"
-              class="rounded-md border border-secondary-300 shadow-sm h-96 bg-vs-dark"
+              class={editor_height(f, @id)}
```

Comment:
We will need to refactor this so we don't call functions, as per my notes yesterday.